docs/code/MixingITKAndNumPy.py

```diff
@@ -14,34 +14,40 @@
 # Run filters on itk.Image
```

Comment:
In other comments class and method names have been surrounded by inverted comas. For the sake of consistency, I'd try to use that notation (or remove them from they were used), e.g. `itk.Image`.

Reply:
Great observation -- I have removed the backticks from the other locations.